version_control/katalyst_by_codurance/css/elements/_colors.css

```diff
@@ -10,6 +10,12 @@
     --light-green: hsla(175, 85%, 95%, 1);
     --dark-blue: hsla(226, 56%, 11%, 1);
 
+    --grey-100: hsla(0, 0%, 90%);
+    --grey-300: hsla(0, 0%, 70%);
+    --grey-500: hsla(0, 0%, 50%);
+    --grey-700: hsla(0, 0%, 30%);
+    --grey-900: hsla(0, 0%, 10%);
```

Comment:
Are all these variants of grey going to be used? I only found grey-100 and grey-300 used for now on the rest of the code 🤔

Reply:
On the XD file there isn't any reference to the different types of gray we have at the moment. Actually we have several with different tints on it. I thought having this scale could help us to use one or another depending on the situation instead of having just 2.